### DISCUSSION TITLE
Extracting data from ArchR

### DISCUSSION THREAD

**Discussion by dimurali93** [GreenleafLab/ArchR#1803] [Questions / Documentation] [answered]
At the end of my ArchR analysis is there a way to export sn-ATAC matrix, along with metadata in a cisTopic object ?

**Comment by rcorces** [comment, marked as answer]
No. You can of course export the matrix and metadata and form them into a cisTopic object yourself though.
Fair warning that depending on what matrix you try to export and how many cells are in your project, this may not work due to limits on vector size in R.
https://www.archrproject.com/reference/getMatrixFromProject.html
